Content.Client/Administration/UI/BwoinkWindow.xaml

```diff
@@ -1,6 +1,9 @@
 ﻿<SS14Window
     xmlns="https://spacestation14.io"
-    SetSize="400 200">
+    SetSize="400 200"
+    HeaderClass="windowHeaderAlert"
```

Comment:
Got errors about this line. Might need to be just Class.

Reply:
It needed an engine update, which was pushed a few hours ago.